### PR TITLE
Add token transform workflow to share styling with support.forcepoint.com project

### PR DESCRIPTION
**Pull request by callinmullaney**
# Ticket(s)

- [FP-680: Design token transforms or custom properties import](https://app.clickup.com/t/36718269/FP-680)

## Purpose

- Implement a project build workflow that converts a json file of design tokens to `tailwind-theme-preset.js`.
- The values of the existing `tailwind-theme-preset.js` were converted to json and stored in a separate repository: https://github.com/fourkitchens/fp-design-tokens - This was done so the `support.forcepoint.com` salesforce project can use these values as it does not utilize Tailwind or React

### Functional Testing

- [ ] Checkout this branch and run `npm install`. Verify the `@fourkitchens/fp-design-tokens` package is now in `node_modules`
- [ ] Verify after running `npm install` there is now a `tailwind-theme-preset.js` added to the root of the project
- [ ] Run `npm run dev`
- [ ] Verify storybook starts and components build properly. 
- [ ] Review an given component and verify the preset values are being loaded (fonts, colors, spacing, ...etc)

### Readme Updates

- (optional) Describes changes made to the project's README file

### Notes

- (optional) Document any intentionally unfinished parts or known issues within this PR

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-680 Design token transforms or custom properties import](https://app.clickup.com/t/36718269/FP-680)

File: .gitignore [hunk 1]
```diff
@@ -26,5 +26,8 @@ dist-ssr
 # Storybook
 storybook-static
 
+# tranformed theme preset tokens for tailwind
+tailwind-theme-preset.js
```
Comment (callinmullaney) [inline .gitignore:30]: Maybe this file shouldn't be ignored?

**Comment by callinmullaney** [comment]
@mcortes19 I'm assigning this PR to you as you seem the best person to review given your understanding of the project as a whole. In short, I've extracted the values of the existing `tailwind-theme-preset.js` file and create a new json file hosted in another repository. The changes in this PR consume this JSON file and generate the `tailwind-theme-preset.js` file.

I created this PR/repository as one of my goals for the support.forcepoint.com salesforce project is to keep our efforts with the this project in sync. I realize this JSON file isn't a direct export of figma tokens but it does ensure if these values are adjusted it will affect both projects. 

Let me know if you have any thoughts or feedback. Happy to zoom over this as well.

**Comment by gstvdesign** [comment]
PR not needed.

**Pull request closed without merging**